packages/server/server.py

```
@@ -0,0 +1,249 @@
+        with open(path, 'r') as ro:
+            data = json.load(ro)
+            response_data = dict(
+                info_dict=dict(id=data.get("info_dict", {}).get("id")),
+                status=data.get("status", "downloading"),
+                speed=data.get("speed", 0),
+                downloaded_bytes=data.get("downloaded_bytes", 0),
+                total_bytes=data.get("total_bytes", 1),
+                elapsed=data.get("elapsed", 0),
+            )
+            return flask.jsonify(error=False, data=response_data), 200
+    except json.JSONDecodeError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)